BackEnd/BackEndServer/Views/Alert/Dashboard.cshtml

```diff
@@ -1,13 +1,13 @@
 ﻿@using BackEndServer.Models.Enums
 @using BackEndServer.Models.ViewModels
 @model AlertDashboardInformation
-@{ Html.RenderPartial("~/Views/Alert/AlertCreationModal.cshtml", Model.Availablecameras); }
-@{ Html.RenderPartial("~/Views/Alert/AlertDeletionConfirmationModal.cshtml"); }
-<h1 class="page-title center-align">Your Alerts</h1>
+@{ Html.RenderPartial("../Alert/AlertCreationModal", Model.Availablecameras); }
+@{ Html.RenderPartial("AlertDeletionConfirmationModal"); }
```

Comment:
.cshtml partial style should be kept. The other style causes issues on a deployment environment